### ISSUE TITLE
Pipe will never be connected in debug mode

### ISSUE THREAD

**Issue by uriyay** [ivanfratric/winafl#101]
If you run afl-fuzz with "-debug" you will never be able to connect to the named pipe.
afl-fuzz.c will try to ConnectNamedPipe in create_target_process().

but winafl.dll will:
```
if(!options.debug_mode) {
    setup_pipe();
    setup_shmem();
} else {
    winafl_data.afl_area = (unsigned char *)dr_global_alloc(MAP_SIZE);
}
```
(in dr_client_main, winafl.c)
	
so if you pass "-debug" - winafl will never connect the pipe. And that happens to cause me timeouts and "cannot kill process -1" message.
Thats because afl-fuzz will extract the pid after ConnectNamedPipe will finish, so I guess what happens is that meanwhile another thread realize there is timeout and try to kill it with the global var child_pid which is still "-1".

**Comment by ivanfratric** [comment]
Correct, with -debug flag, winafl.dll will never connect to the pipe.
the -debug flag is only meant to be used with drrun.exe and should never be used in combination with afl-fuzz.
